tests/contrib/sensors/test_redis_sensor.py

```diff
@@ -47,22 +47,25 @@ def setUp(self):
             key='test_key'
         )
 
-    @patch("airflow.contrib.hooks.redis_hook.RedisHook.key_exists")
-    def test_poke(self, key_exists):
-        key_exists.return_value = True
-        self.assertTrue(self.sensor.poke(None))
+    @patch("airflow.contrib.sensors.redis_key_sensor.RedisHook")
+    def test_poke(self, RedisHook):
+        RedisHook.return_value.get_conn.return_value.exists.side_effect = [True, False]
+        self.assertTrue(self.sensor.poke(None), "Key exists on first call.")
+        RedisHook.assert_called_once_with('redis_default')
+        RedisHook.return_value.get_conn.assert_called_once_with()
+        RedisHook.return_value.get_conn.return_value.exists.assert_called_once_with('test_key')
+        self.assertFalse(self.sensor.poke(None), "Key does NOT exists on second call.")
 
-        key_exists.return_value = False
-        self.assertFalse(self.sensor.poke(None))
-
-    @patch("airflow.contrib.hooks.redis_hook.StrictRedis.exists")
-    def test_existing_key_called(self, redis_client_exists):
+    @patch("airflow.contrib.hooks.redis_hook.StrictRedis")
+    @patch('airflow.contrib.hooks.redis_hook.RedisHook.get_connection')
```

Comment:
We should not need to patches anymore, since we have Redis running.

Reply:
Why do we want to test redis in tests for airflow project? I understand that we may need some integration tests (and the best would be to have them separate to unit test) but it does not mean we need all of it to connect everywhere all the time :) That focus of integration, is the reason that current ticket event exist. AIRFLOW-999 had integration tests, but no proper unit tests.

Reply:
That would be my preference. We don't have that many Redis tests anyway.

Reply:
OK. I will go with your desire, but it would be nice if you could bring any justification in the future :)